prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div>
+    <a href="http://127.0.0.1:5500/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
Rather than use the absolute URL to the `index.html` of your Live Server, use a relative path to the `index.html` in the parent directory (_i.e._ `href="../index.html"`). 

That way, the link works on computers other than your own.